### PR TITLE
Wes/sophtron integration tests

### PR DESCRIPTION
**Pull request by wesrisenmay-mx**

### CODE REVIEW

File: apps/server/package.json [hunk 1]
```diff
@@ -71,6 +71,7 @@
     "@types/jest": "^29.5.12",
     "@typescript-eslint/eslint-plugin": "^6.20.0",
     "babel-plugin-js-logger": "^1.0.17",
+    "cross-env": "^7.0.3",
```
Comment (tphalp) [inline apps/server/package.json:74]: I feel like this could be a devDep, since it's only being used for tests.

**Review by tphalp — APPROVED** [review]
One simple comment about `crossenv`

File: apps/server/package.json (hunk 1 above)
Comment (wesrisenmay-mx) [reply apps/server/package.json:74]: It is a dev dependency, but where it collapsed this file sure made it look like it was a regular dependency.

**Review by Craiting — APPROVED** [review]
Excellent work

File: apps/server/package.json (hunk 1 above)
Comment (tphalp) [reply apps/server/package.json:74]: Oh got it. Haha.

**Pull request merged**